### DISCUSSION TITLE
After installing 9.22.4 Directus shows 9.22.3 in Settings

### DISCUSSION THREAD

**Discussion by Pachat** [directus/directus#17133] [Archive]
Here is the `package.json` file
```
...
  "dependencies": {
    "create-directus-extension": "^9.22.3",
    "directus": "^9.22.4",
    "graphql": "^16.6.0",
    "pg": "^8.8.0"
  },
...
```
Directus shows `9.22.3` instead of `9.22.4` in the Settings in `#navigation`
after updating from `9.21.0` with 

1. `npm install directus --save`
2. `npm install create-directus-extension --save`
3. `npm install graphql --save`

**Comment by rijkvanzanten** [comment]
Duplicate of https://github.com/directus/directus/issues/17115
